### ISSUE TITLE
Difference between -o and -s ?

### ISSUE THREAD

**Issue by tashrifbillah** [MIC-DKFZ/HD-BET#37]
Folks, can you remind us the difference between `-o` as a file and `-s`? I used:

> hd-bet -i T1w.nii.gz -o mask.nii.gz

but got two files:

> mask.nii.gz and mask_mask.nii.gz

Do I need:

> hd-bet -i T1w.nii.gz -o mask.nii.gz -s 0

**Comment by FabianIsensee** [comment]
-o is the output image (your input image with everything that is not brain set to 0)
-s is the segmentation mask of the brain, so it's 1 wherever the model thinks the brain is

**Comment by tashrifbillah** [comment]
Yes, I figured them out through experiement. I'll try to add them to your README.md and submit a PR.

**Comment by FabianIsensee** [comment]
have you tried `hd-bet -h`? That should tell you all this as well ;-)

**Comment by tashrifbillah** [comment]
It did not!

**Comment by FabianIsensee** [comment]
I see :-) If you want to make a PR clarifying this I am happy to accept it :-)
